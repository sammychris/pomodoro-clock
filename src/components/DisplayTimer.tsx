import React from "react";

const DisplayTimer = ({ text, display }) => {
  return (
    <div id="display">
      <div id="timer-label">{text}</div>
      <div id="time-left">{display}</div>
    </div>
  );
};

export default DisplayTimer;
