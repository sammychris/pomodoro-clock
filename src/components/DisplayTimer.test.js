import { shallow } from "enzyme";
import DisplayTimer from "./DisplayTimer";

const wrapper = shallow(<DisplayTimer />);

it("expect to render display-timer component", () => {
  expect(wrapper.length).toEqual(1);
});

it("expect to take snap shot of display-timer component", () => {
  expect(wrapper).toMatchSnapshot();
});
