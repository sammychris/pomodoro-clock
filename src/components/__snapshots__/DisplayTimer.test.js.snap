// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to take snap shot of display-timer component 1`] = `ShallowWrapper {}`;
